x-pack/examples/reporting_example/public/components/app.tsx

```diff
@@ -33,24 +33,24 @@ import { NavigationPublicPluginStart } from '../../../../../src/plugins/navigati
 import { constants, ReportingStart } from '../../../../../x-pack/plugins/reporting/public';
 import { JobParamsPDF } from '../../../../plugins/reporting/server/export_types/printable_pdf/types';
 
-interface ReportingExampleAppDeps {
+interface ReportingExampleAppProps {
   basename: string;
   notifications: CoreStart['notifications'];
   http: CoreStart['http'];
   navigation: NavigationPublicPluginStart;
   reporting: ReportingStart;
   screenshotMode: ScreenshotModePluginSetup;
+  version: string;
```

Comment:
the way i was hoping for this to work is that apps don't need to do anything special.

report creation always go thru our share plugin and we can attach the version to the params, it shouldn't be needed by the app.

this means that example app (and canvas, dashboard, ...) shouldn't change at all.